documentation/modules/upgrading/con-downgrade-target-version.adoc

```diff
@@ -2,17 +2,42 @@
 //
 // assembly-downgrading-kafka-versions.adoc
 
-[id='com-target-downgrade-version-{context}']
+[id='con-target-downgrade-version-{context}']
 
-= Target downgrade version
+= Kafka version compatibility for downgrades
 
-How the Cluster Operator handles a downgrade operation depends on the `log.message.format.version`.
+Kafka downgrades are dependent on compatible current and target xref:ref-kafka-versions-{context}[Kafka versions],
+and the state at which messages have been logged.
 
-* If the target downgrade version of Kafka has the same `log.message.format.version` as the current version, the Cluster Operator downgrades by performing a single rolling restart of the brokers.
-* If the target downgrade version of Kafka has a different `log.message.format.version`, downgrading is only possible if the running cluster has _always_ had `log.message.format.version` set to the version used by the downgraded version.
-+
+You cannot revert to the previous Kafka version if the `inter.broker.protocol.version` differs, or messages have been added
+to message logs that use a newer `log.message.format.version`.
+
+The `inter.broker.protocol.version` determines the schemas used for persistent metadata stored by the broker, such as the schema for messages written to `__consumer_offsets`.
+If you downgrade to a version of Kafka that uses an earlier `inter.broker.protocol.version`, the cluster will not work because it will not understand the messages.
```

Comment:
An "earlier IBP" isn't quite the right language. Each version of Kafka supports a range of IBPs, so "earlier" is a little ambiguous (it could be taken to mean earlier in the range of IBPs supported by a particular version). 

```suggestion
If you downgrade to a version of Kafka that doesn't understand an `inter.broker.protocol.version` that has (ever) been previously used in the cluster the broker will encounter data they cannot understand.
```